### PR TITLE
7 In your user area write a small nodejs program

### PR DESCRIPTION
**Pull request by nisharai26**

### CODE REVIEW

File: nisharai26/exampleIf.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+function testPerson(age){
+if(age>18){
```
Comment (nmunro) [inline nisharai26/exampleIf.js:2]: Would this be better with `>=` since if someone IS 18 they will still be treated as a child

File: README.md [hunk 2]
```diff
@@ -16,6 +15,8 @@ Philip Edwards [github](https://github.com/phiddle) to see my GitHub profile [we
 Joe Foster - [Click here](https://github.com/JoeFoster-cn) to see my GitHub profile and [click here](https://joefoster-cn.github.io/) to see my GitHub Pages.
 Nathan Robinson [AboutMe](https://github.com/NathanRobinson11/)[GitHub](https://github.com/NathanRobinson11/)
 Lisa Rawlings -[Click here](https://github.com/lisarawlings) to see my GitHub profile
+Stephen Booth -[Github Profile](https://github.com/boothscript) -[Portfolio](https://boothscript.github.io/portfolio2)
```
Comment (nmunro) [inline README.md:18]: Can we remove this line as Stephen is duplicated here.

**Review by nmunro — CHANGES_REQUESTED** [review]

File: nisharai26/exampleIf.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
 function testPerson(age){
-if(age>18){
+if(age>=18){
```
Comment (nmunro) [inline nisharai26/exampleIf.js:2]: Great, thanks for fixing that, all looks good now!

**Review by nmunro — APPROVED** [review]

**Review by Melia-create — APPROVED** [review]

**Review by jadavey91 — APPROVED** [review]
@nmunro doing all the heavy lifting for us! Thanks.

**Pull request merged**